zeppelin-web/src/app/notebook/notebook.controller.js

```diff
@@ -752,6 +753,36 @@
       }
     };
 
+    $scope.setIamOwner = function() {
+      if ($scope.permissions.owners.length > 0 &&
+          _.indexOf($scope.permissions.owners, $rootScope.ticket.principal) < 0) {
+        $scope.isOwner = false;
+        return false;
+      }
+      $scope.isOwner = true;
+      return true;
+    };
+
+    $scope.toggleNotePersonalizedMode = function() {
+      var personalizedMode = $scope.note.config.personalizedMode;
+      if ($scope.isOwner) {
+        BootstrapDialog.confirm({
+          closable: true,
+          title: 'Setting the result display',
+          message: 'Do you want to personalize your analysis??',
```

Comment:
Seems this message is shown in both cases "Personalized mode" and "Shared mode". I think when we toggle this button from "Personalized mode" to "Share mode", then it should be "Do you want to share your analysis?"